spacemacs/config.el

```diff
@@ -168,6 +168,12 @@ Can be installed with `brew install trash'."
 ;; Single space between sentencs is more widespread than double
 (setq-default sentence-end-double-space nil)
 
+;; The C-d rebinding that most shell-like buffers inherit from
+;; comint-mode assumes non-evil configuration with its
+;; `comint-delchar-or-maybe-eof' function, so we disable it
+(with-eval-after-load 'comint
```

Comment:
`with-eval-after-load` is not supported in 24.3 so we have to use `eval-after-load`